CreateDB

```diff
@@ -49,5 +54,5 @@ CREATE TABLE Contacts(
 CREATE TABLE Hashtags(
     hashtag varchar(50) not null,
     uid Integer REFERENCES Users(uid),
-    pid Integer references Parcels(pid)
+    mid Integer REFERENCES Messages(mid)
 );
```

Comment:
Missing primary key

Reply:
Primary key isn't needed, it's a many to many table